panel/src/components/Views/FileView.vue

```diff
@@ -1,9 +1,14 @@
 <template>
   <k-inside :lock="lock">
-    <div class="k-file-view">
+    <div
+      :data-locked="isLocked"
+      :data-id="model.id"
+      :data-role="blueprint"
```

Comment:
Why is the blueprint name used for data-role? Should probably be data-template, right?

Reply:
Yep!